### PR TITLE
Remove cb_printf from rzbin and rawstr

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Dropping `cb_printf` and `rawstr` from rz_bin.
- `izzz` has been dropped because the behavior was the same as `izz` (but worse since it was using `cb_printf`)

### CODE REVIEW

**Review by ret2libc — COMMENTED** [review]
Gonna review later but... Thank you ❤️

**Review by XVilka — CHANGES_REQUESTED** [review]
Code-wise changes look good. But with 42 failing tests careful work is needed: some tests are better with the new output, some look like regressions.

File: librz/main/rz-bin.c [hunk 1]
```diff
@@ -1184,9 +1220,29 @@ RZ_API int rz_main_rz_bin(int argc, const char **argv) {
 	if (baddr != UT64_MAX) {
 		rz_bin_set_baddr(bin, baddr);
 	}
-	if (rawstr == 2) {
-		bf->strmode = rad;
-		rz_bin_dump_strings(bf, bin->minstrlen, bf->rawstr);
+	if (rawstr) {
+		PJ *pj = NULL;
+		if (rad == RZ_MODE_JSON) {
```
Comment (XVilka) [inline librz/main/rz-bin.c]: Please rename this variable.

**Review by XVilka — APPROVED** [review]
Love this change and new results. Amazing work! 🥇

File: test/db/cmd/cmd_i [hunk 2]
```diff
@@ -4342,17 +4325,6 @@ EXPECT=<<EOF
 EOF
 RUN
 
-NAME=izzzj
```
Comment (ret2libc) [inline test/db/cmd/cmd_i:4345]: is there a test for the json output elsewhere?

File: test/db/abi/compilers/pelles_c [hunk 3]
```diff
@@ -26,7 +26,7 @@ EXPECT=<<EOF
    38 * relocs
     4 * sections
     0 * segments
-  117 * strings
+  173 * strings
```
Comment (ret2libc) [inline test/db/abi/compilers/pelles_c:29]: Running rz-bin -zz here it seems that the new PR finds less strings.

**Review by ret2libc — CHANGES_REQUESTED** [review]

File: test/db/cmd/cmd_i (hunk 2 above)
Comment (wargio) [reply test/db/cmd/cmd_i:4345]: yes, there is izzj

File: test/db/abi/compilers/pelles_c (hunk 3 above)
Comment (wargio) [reply test/db/abi/compilers/pelles_c:29]: where?
image

**Pull request merged**